Website/src/rendering/src/stories/PreviewSearch/SearchInput.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import SearchInput from '../../components/PreviewSearch/SearchInput';
+import { mockDiscoverData } from '../mock-discover-data';
+
+export default {
+  title: 'Components/PreviewSearch/SearchInput',
+  component: SearchInput,
+} as ComponentMeta<typeof SearchInput>;
+
+const Template: ComponentStory<typeof SearchInput> = (args) => <SearchInput {...args} />;
+
+export const Default = Template.bind({});
+Default.args = mockDiscoverData.searchInputProps;
```

Comment:
Currently there are no styles attached to the component which makes the story more or less pointless. I'd suggest using decorators to add the neccessary wrapping classes for the styles to be applied. While you're at it you can do the same for the PreviewSearch story as well. This should do it:
```
Default.decorators = [
  (Story) => (
    <div className="shop-navigation">
      <div className="shop-navigation-content">
        <div className="shop-search-input-container">
          <Story />
        </div>
      </div>
    </div>
  ),
];